### PR TITLE
collector-redshift

### PR DESCRIPTION
**Pull request by kunalraj1705**
redshift collector script

### CODE REVIEW

File: src/collectors/aws/redshift/ClusterSecurityGroups.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import * as AWS from "aws-sdk";
```
Comment (paragpoddar) [inline src/collectors/aws/redshift/ClusterSecurityGroups.ts]: Please follow other file name. First letter should be small.

**Comment by paragpoddar** [comment]
When you are creating new file it should be exported from index.

File: src/collectors/aws/redshift/ClusterSecurityGroups.ts (hunk 1 above)
Comment (paragpoddar) [reply src/collectors/aws/redshift/ClusterSecurityGroups.ts]: ClusterSecurityGroups.ts --> clusterSecurityGroups.ts

Comment (kunalraj1705) [reply src/collectors/aws/redshift/ClusterSecurityGroups.ts]: The necessary changes have been done. Please have a look at the code.

Comment (Marina9627) [reply src/collectors/aws/redshift/ClusterSecurityGroups.ts]: Please check the modified versions of the codes.

File: src/collectors/aws/redshift/index.ts [hunk 2]
```diff
@@ -1,2 +1,8 @@
 export * from "./audit_logs";
 export * from "./clusters";
+export * from "./reserved_nodes";
```
Comment (paragpoddar) [inline src/collectors/aws/redshift/index.ts]: There is no file called "reserved_nodes". Getting error for this.

Comment (kunalraj1705) [reply src/collectors/aws/redshift/index.ts]: The updated "index.ts" file has be uploaded, please have a look at it.

**Pull request merged**